### PR TITLE
Regenerated Storage and Compute client using latest AutoRest

### PR DESCRIPTION
**Pull request by stankovski**

### CODE REVIEW

File: tools/autorest.gen.cmd [hunk 1]
```diff
@@ -12,6 +12,6 @@ set autoRestExe=%repoRoot%\packages\autorest.%autoRestVersion%\tools\AutoRest.ex
 %repoRoot%\tools\nuget.exe install autorest %source% -Version %autoRestVersion% -o %repoRoot%\packages -verbosity quiet
 
 @echo on
-%autoRestExe% -Modeler Swagger -CodeGenerator Azure.CSharp -Namespace %namespace% -Input %specFile% -outputDirectory %generateFolder% -Header MICROSOFT_MIT
+%autoRestExe% -Modeler Swagger -CodeGenerator Azure.CSharp -Namespace %namespace% -Input %specFile% -outputDirectory %generateFolder% -Header MICROSOFT_MIT %~5
```
Comment (huangpf) [inline tools/autorest.gen.cmd]: What does ``` %~5``` do?

File: src/resourcemanagement/compute/compute.tests/VMScaleSetTests/VMScaleSetOperationalTests.cs [hunk 2]
```diff
@@ -132,18 +132,15 @@ public void TestVMScaleSetBatchOperations()
...
+                    m_CrpClient.VirtualMachineScaleSets.Start(rgName, vmScaleSet.Name, virtualMachineScaleSetInstanceIDs);
+                    virtualMachineScaleSetInstanceIDs = new List<string>() { "0" };
+                    m_CrpClient.VirtualMachineScaleSets.PowerOff(rgName, vmScaleSet.Name, virtualMachineScaleSetInstanceIDs);
                     m_CrpClient.VirtualMachineScaleSets.UpdateInstances(rgName, vmScaleSet.Name, virtualMachineScaleSetInstanceIDs);
-                    virtualMachineScaleSetInstanceIDs.InstanceIds = new List<string>() { "1" };
-                    m_CrpClient.VirtualMachineScaleSets.RestartInstances(rgName, vmScaleSet.Name, virtualMachineScaleSetInstanceIDs);
-                    m_CrpClient.VirtualMachineScaleSets.DeallocateInstances(rgName, vmScaleSet.Name, virtualMachineScaleSetInstanceIDs);
+                    virtualMachineScaleSetInstanceIDs = new List<string>() { "1" };
+                    m_CrpClient.VirtualMachineScaleSets.Restart(rgName, vmScaleSet.Name, virtualMachineScaleSetInstanceIDs);
+                    m_CrpClient.VirtualMachineScaleSets.Deallocate(rgName, vmScaleSet.Name, virtualMachineScaleSetInstanceIDs);
```
Comment (huangpf) [inline src/resourcemanagement/compute/compute.tests/VMScaleSetTests/VMScaleSetOperationalTests.cs]: What's the signature for deallocating the entire VMSS? ```Deallocate(rgName, vmssName)``` or ```Deallocate(rgName, vmssName, null)```?

File: src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineScaleSetsOperationsExtensions.cs [hunk 3]
```diff
@@ -767,12 +767,12 @@ public static void StartInstances(this IVirtualMachineScaleSetsOperations operat
             /// <param name='vmScaleSetName'>
             /// The name of the virtual machine scale set.
             /// </param>
-            /// <param name='vmInstanceIDs'>
-            /// The list of virtual machine scale set instance IDs.
+            /// <param name='instanceIds'>
+            /// Gets or sets the virtual machine scale set instance ids.
             /// </param>
-            public static void BeginStartInstances(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName, VirtualMachineScaleSetVMInstanceIDs vmInstanceIDs)
+            public static void BeginStart(this IVirtualMachineScaleSetsOperations operations, string resourceGroupName, string vmScaleSetName, IList<string> instanceIds = default(IList<string>))
```
Comment (huangpf) [inline src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: It seems by default it's an empty list, so the request body is most likely ```{}```. Not sure whether this works or not... will give it a try in test re-recording.

Comment (huangpf) [inline src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: Can the default be ```null```?

**Comment by huangpf** [comment]
LGTM.

File: tools/autorest.gen.cmd (hunk 1 above)
Comment (stankovski) [inline tools/autorest.gen.cmd]: Take the 5th parameter and strip quotes :)

File: src/resourcemanagement/compute/compute.tests/VMScaleSetTests/VMScaleSetOperationalTests.cs (hunk 2 above)
Comment (stankovski) [inline src/resourcemanagement/compute/compute.tests/VMScaleSetTests/VMScaleSetOperationalTests.cs]: `Deallocate(rgName, vmssName)`

File: src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineScaleSetsOperationsExtensions.cs (hunk 3 above)
Comment (stankovski) [inline src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: it is null

File: src/resourcemanagement/compute/compute.tests/VMScaleSetTests/VMScaleSetOperationalTests.cs (hunk 2 above)
Comment (huangpf) [inline src/resourcemanagement/compute/compute.tests/VMScaleSetTests/VMScaleSetOperationalTests.cs]: Thanks~

File: src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineScaleSetsOperationsExtensions.cs (hunk 3 above)
Comment (huangpf) [inline src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineScaleSetsOperationsExtensions.cs]: Thanks~

**Pull request merged**